README.md

```diff
@@ -0,0 +1,122 @@
+
+### Using GitHub in RStudio
+To clone this repository onto your local environment, follow these steps:
+1. On GitHub, navigate to the Code tab of the repository.
+2. On the right side of the screen, click `Clone or download`.
+3. Click the `Copy to clipboard` icon to the right of the repository URL.
+4. Open RStudio on your local environment.
+5. Click `File`, `New Project`, `Version Control`, `Git`.
+6. Paste the repository URL and enter TAB to move to the Project directory name field.
+7. Click `Create Project`.
+8. Finish reading the README.md file
+
+### Further Resources
+The above will get your basic setup ready but here are some links if you are interested in reading a bit further.
+
+- How to Use RStudio:
+  - http://www.rstudio.com/ide/docs/
+- Getting Started with GitHub
+  - https://guides.github.com/activities/hello-world/
+- More RStudio Support:
+  - https://support.rstudio.com/hc/en-us
+- Using Version Control with RStudio
+  - http://www.rstudio.com/ide/docs/version_control/overview
+- R Installation and Administration
+  - http://cran.r-project.org/doc/manuals/R-admin.html
+- R FAQ:
+  - http://cran.r-project.org/doc/FAQ/R-FAQ.html
+- More about add-on packages in the R Installation and Administration Manual
+  - http://cran.r-project.org/doc/manuals/R-admin.html#Add_002don-packages
+  
+## Repository Organization
+The repository is organized into the following structure (as suggested [here](https://jhuadvdatasci.substack.com/p/jhu-ads-2020-week-3-organizing-a)):
+- README.md
+- .gitignore
+- Bus Stop Census.Rproj
+- .RProfile
+- data/
+  - raw_data/
+  - tidy_data/
+  - codebook.md
+- code/
+  - raw_code/
+  - final_code/
+- figures/
+  - exploratory_figures/
+  - explanatory_figures/
+- products/
+  - writing
+  - presentations
+ 
+ The following subsections contain descriptions of the main components in this file structure.
+ 
+### README.md
+This is what you are reading now. It contains everything you need to know to understand the contents of the repository and how to use it.
+  
+### Bus Stop Census.Rproj
+This is the R project file for the Bus Stop Census analysis. An R Project is simply a working directory designated with a .RProj file. When you open a project (using File/Open Project in RStudio or by double–clicking on the .Rproj file outside of R), the working directory will automatically be set to the directory that the .RProj file is located in.
+
+### .RProfile
+This is a script that runs every time R starts. The `.RProfile` included in this repository will run each time the `Bus Stop Census.RProj` file is opened. The project `.RProfile` script is important because it installs and loads the libraries needed to run the code as well as the GTFS data that is the key to linking bus stop IDs with their associated route, coordinate, and schedule information.
+
```

Comment:
Does RStudio automatically install any required packages or do they need to be manually installed first? If a manual installation of packages is necessary, please provide instructions for that.

Reply:
I will update the .RProfile to both install and load the necessary packages. By doing this, the users will not have to fiddle with manually installing them